k8s/lib.sh

```diff
@@ -31,6 +31,29 @@ k8s::common::setup_env() {
   _K8S_ENV_SETUP_ONCE="1"
 }
 
+# Cleanup configuration left by the network component
+#   - Iptables Rules
+#   - Network Interfaces
+#   - Traffic Control(tc) rules
+k8s::remove::network() {
```

Comment:
I understand that you do not want to mention Cilium in the signature as CNIs might change. But this removal process will always be somewhat CNI specific so it makes sense to add some context.

Could you add some links/explanations where this comes from (e.g. codepointers to cilium cleanup etc)? Thanks!